openshift/.pipeline/Jenkinsfile

```diff
@@ -146,19 +170,29 @@ pipeline {
          -p APPLICATION_DOMAIN=${SUBDOMAIN}-test.${DOMAIN} \
          -p MEMORY_LIMIT=${RUN_MEMORY} \
          -p NAMESPACE=${TOOLS_NAMESPACE} \
-         -p TAG=dev | oc apply -f - -n ${TEST_NAMESPACE}"
+         -p TAG=test | oc apply -f - -n ${TEST_NAMESPACE}"
       sh "oc tag ${DYNAMICS_ADAPTER}:dev ${DYNAMICS_ADAPTER}:test -n ${TOOLS_NAMESPACE}"
       // WaitForDeploy(DYNAMICS_ADAPTER, DEV_NAMESPACE)
      },
-     'Sample Adapter': {
+     'SampleAdapter': {
       sh "oc process -f openshift/templates/sample-adapter.dc.yaml \
          -p NAME=${SAMPLE_ADAPTER} \
          -p APPLICATION_DOMAIN=${SUBDOMAIN}-test.${DOMAIN} \
          -p MEMORY_LIMIT=${RUN_MEMORY} \
          -p NAMESPACE=${TOOLS_NAMESPACE} \
-         -p TAG=dev | oc apply -f - -n ${TEST_NAMESPACE}"
+         -p TAG=test | oc apply -f - -n ${TEST_NAMESPACE}"
       sh "oc tag ${SAMPLE_ADAPTER}:dev ${SAMPLE_ADAPTER}:test -n ${TOOLS_NAMESPACE}"
       // WaitForDeploy(ICBC_ADAPTER, DEV_NAMESPACE)
+     },
+     'ICBCAdapter': {
+      sh "oc process -f openshift/templates/icbc-adapter.dc.yaml \
+         -p NAME=${ICBC_ADAPTER} \
+         -p APPLICATION_DOMAIN=${SUBDOMAIN}-test.${DOMAIN} \
+         -p MEMORY_LIMIT=${RUN_MEMORY} \
+         -p NAMESPACE=${TOOLS_NAMESPACE} \
+         -p TAG=test | oc apply -f - -n ${TEST_NAMESPACE}"
+      sh "oc tag ${ICBC_ADAPTER}:dev ${ICBC_ADAPTER}:test -n ${TOOLS_NAMESPACE}"
+      // WaitForDeploy(ICBC_ADAPTER, DEV_NAMESPACE)
```

Comment:
Was wondering why these comments are included if they're never used?

Reply:
Good point, we can uncomment these